AnimalCompetition/Main.java

```diff
@@ -113,239 +118,280 @@ else if (humanOrAI == 1) {
 
 
 
-    
-    for (int i = 1; i <= 2; i++) { //allows for each player to get a turn buying equipment each round
-      while (moveAroundTown != 4) { //4 exits the town and lets player2 decide what to buy
-        System.out.println("Where would you like to go?");
-        System.out.println("1. Arivanya's Animals, an animal breeder.");
-        System.out.println("2. Bothela's Animal Hospital, an animal healer.");
-        System.out.println("3. Balimund's Forge, a weapon and armour seller.");
-        System.out.println("4. Leave town, and head to the fight.");
-        moveAroundTown = scanner.nextInt();
-        
-        //allows for all the operations of an animal store
-        while (moveAroundTown == 1) { 
-          System.out.println("Welcome to Arivanya's Animals. We have all sorts of animals for purchase. So what animal would you like to buy?");
-          System.out.println("1. Animals");
-          System.out.println("2. Leave Arivanya's");
-          moveInShops = scanner.nextInt(); 
-          if (moveInShops == 1) { 
-            for (var animal: animals) { //prints all the animals and their stats
-              System.out.print(String.format("I'm a %s. ", animal.species()));
-              var fighter = (IFightable)animal;
-              if (fighter.canAttack()) {
-                System.out.print(String.format("I do %s damage. ", fighter.attackPoints()));
-              } else {
-                System.out.print("I can't attack. "); 
+    while (player1money > lowestAnimalCost && player2money > lowestAnimalCost) {
```

Comment:
This main function is waaaaaaaayyyyyyyyyyyyyyyy too long.  Break it up into functional parts and put them into functions.  Otherwise your future-self will hate you.  If a function body spans more than a page of code it becomes harder to read and follow.  It also makes the code quite delicate because you have be sure to line up all the code-block parenthesis and brackets.